_posts/events/2025-09-08-workshop-create-python-package-2hour.md

```diff
@@ -0,0 +1,112 @@
+
+#### Part 3: Publish and share your Python package (30 minutes)
+
+- <i class="fas fa-upload" style="color: #999;"></i> **PyPI Publishing:** Get your package on the Python Package Index
+- <i class="fas fa-shield-alt" style="color: #999;"></i> **Security Best Practices:** Protect your package and users when you are publishing using Trusted Publishing.
+- <i class="fas fa-sync-alt" style="color: #999;"></i> **Automation Setup:** Never manually release your package again; instead use GitHub actions with trusted publishing.
+
+#### Part 4: Documentation for a healthy project (20 minutes)
+
+- <i class="fas fa-users" style="color: #999;"></i> **What docs matter most?:** A brief overview of documentation to make it easier to onboard both contributors and new maintainers to your project.
+
+
+#### Wrap up: Questions (20 Minutes)
+
+- <i class="fas fa-question-circle" style="color: #999;"></i> **Q&A:** Get your questions answered
+
+### Why attend
+
+- <i class="fa-solid fa-rocket" style="color: #999;"></i> Learn modern packaging practices used by successful Python projects
+- <i class="fa-solid fa-tools" style="color: #999;"></i> Get production-ready templates and automation you can use immediately
+- <i class="fa-solid fa-shield-alt" style="color: #999;"></i> Understand publishing security best practices from day one
+- <i class="fa-solid fa-eye" style="color: #999;"></i> See live demos of the complete packaging workflow
+- <i class="fa-solid fa-graduation-cap" style="color: #999;"></i> Learn from pyOpenSci's experience reviewing dozens of packages
+
+### Perfect for you if...
+
+- <i class="fas fa-check" style="color: #81c0aa;"></i> You have Python code that others could benefit from using
+- <i class="fas fa-check" style="color: #81c0aa;"></i> You're tired of sharing code via email or messy file transfers
+- <i class="fas fa-check" style="color: #81c0aa;"></i> You want people to be able to install and use your tools
+- <i class="fas fa-check" style="color: #81c0aa;"></i> You've heard of PyPI but don't know how to get your package there and how to do it securely.
+- <i class="fas fa-check" style="color: #81c0aa;"></i> You want to better understand the complex Python packaging ecosystem
+
+### What you'll walk away with
+
+- **Complete Package Template:** Ready-to-use project structure
+- **Automation Workflows:** GitHub Actions for testing and publishing
+- **Quality Checklists:** Never miss important packaging steps
+- **Publishing Guide:** Step-by-step instructions for secure PyPI deployment using GitHub actions
+
+### Pre-requisites
+
+To successfully follow along in this workshop, you should:
+
+* Know how to write Python code
+* Understand how to write and use functions
+* Have a Free account to create a GitHub account (you can work using GitHub codespaces during this workshop, and setting this account up is free if you don't already have one.
+* You should have internet access
+
+### Your instructors & helpers
```

Comment:
Let's add people here who wish to help with the workshop!! Please either add an inline edit comment to add yourself or add your name and a few sentences about you to this comment here!!